src/test/java/ch/uzh/ifi/hase/soprafs22/service/Integration/GameServiceIntegrationTest.java

```diff
@@ -70,123 +75,164 @@ void setup() {
         redUnitPosition = positionWithTeamUnit(game, Team.RED);
         blueUnitPosition = positionWithTeamUnit(game, Team.BLUE);
         noUnitPosition = positionWithNoUnit(game);
+
+        attackCommand = new AttackCommand();
```

Comment:
Setup creates those commands but nothing is set, maybe you can create them where they are declared (line 53).

Reply:
yep, thx :)
changed